site/styles/_nav.scss

```diff
@@ -340,14 +346,15 @@
     height: auto;
     transition: all .45s ease-in;
     min-height: 100vh;
+    box-shadow: 0 6px 6px rgba(#000, .3); // Light Theme
```

Comment:
Does this need to be themed?

Reply:
Probably shouldn't be unless you want different colour shadows.